cmd/minikube/cmd/start.go

```diff
@@ -1577,7 +1577,7 @@ $ minikube config unset kubernetes-version`)
 
 	if paramVersion == "" || strings.EqualFold(paramVersion, "stable") {
 		paramVersion = constants.DefaultKubernetesVersion
-	} else if strings.EqualFold(paramVersion, "latest") {
+	} else if strings.EqualFold(paramVersion, "latest") || strings.EqualFold(paramVersion, "newest") {
```

Comment:
while we are at it, how about we accept both lower case and upper case

strings.EqualFold(strings.ToLower(paramVersion), "latest") ...(and same for other one)



strings.EqualFold(paramVersion, "latest")

Reply:
@medyagh good idea, I'll PR changes.

Reply:
@medyagh `strings.EqualFold` is case-insensitive already